src/cuda-kernels/manual_awkward_ListArray_num.cu

```diff
@@ -91,12 +80,10 @@ awkward_ListArray64_num_64(
     threads_per_block = dim3(length, 1, 1);
   }
 
-  cuda_ListArray_num<int64_t , int64_t><<<blocks_per_grid, threads_per_block>>>(
-    tonum,
-    fromstarts,
-    fromstops);
+  cuda_ListArray_num<int64_t, int64_t>
+      <<<blocks_per_grid, threads_per_block>>>(tonum, fromstarts, fromstops);
 
   cudaDeviceSynchronize();
 
   return success();
-}
+}
```

Comment:
Can we end the file with an empty line?

Reply:
It's generally good to end all files with an empty line—I hadn't noticed this one. However, I don't want to restart the tests for it, which can take a total of 20 minutes, because I'm waiting for it to do the master → main thing. We can open new PRs to fix things like this immediately afterward.